packages/embark/src/test/modules/solidity/remapImports.js

```diff
@@ -2,7 +2,6 @@
 const { dappPath, File, Types, prepareForCompilation } = require('embark-utils');
 const path = require("path");
 const {expect} = require("chai");
-const fs = require("../../../lib/core/fs");
```

Comment:
There was no usage of `fs` in the entire module as it relies on `fsNode` (one line below), so this could be safely removed.